platform/admin/ui/src/main/webapp/js/views/installer/Configuration.view.js

```diff
@@ -19,28 +19,242 @@ define([
     'icanhaz',
     'underscore',
     'backbone',
+    'jquery',
     'text!templates/installer/configuration.handlebars',
+    'text!templates/installer/configurationItem.handlebars',
+    './Certificate.view.js',
     'modelbinder',
     'perfectscrollbar',
     'multiselect'
-    ], function (Marionette, ich, _, Backbone, configurationTemplate) {
+    ], function (Marionette, ich, _, Backbone, $, configurationTemplate, configurationItemTemplate, CertificateView) {
 
     ich.addTemplate('configurationTemplate', configurationTemplate);
+    ich.addTemplate('configurationItemTemplate', configurationItemTemplate);
 
 
+/*
```

Comment:
These models should not be in the view. They should be moved to js/models/installer